### PR TITLE
feat(git-practice): Fer's rehearsal

### PR DESCRIPTION
**Pull request by mframirezfuentes**
create a new branch with content a index.html and index.js

### CODE REVIEW

File: fer-git-practice/index.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    }else if(a<b){
+
+        return "bes mayor que a"
+    }
+    else{
+        return "son iguales"
+    }
+
+  
+}
```
Comment (arielarmijo) [inline fer-git-practice/index.js]: add a new line in the end of the file please.

File: fer-git-practice/index.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Hola!</h1>
+</body>
+</html>
```
Comment (arielarmijo) [inline fer-git-practice/index.html]: please add new line here.

Comment (RickyCode) [inline fer-git-practice/index.html]: please add a blank line at the end of the file

**Comment by arielarmijo** [comment]
Well done!

File: fer-git-practice/index.js (hunk 1 above)
Comment (RickyCode) [inline fer-git-practice/index.js]: add blank line at the end

**Review by carrizotamara — COMMENTED** [review]
nice job

**Review by arielarmijo — COMMENTED** [review]
nice work

**Review by arielarmijo — DISMISSED** [review]

File: fer-git-practice/index.html (hunk 2 above)
Comment (mframirezfuentes) [reply fer-git-practice/index.html]: finish

File: fer-git-practice/index.js (hunk 1 above)
Comment (mframirezfuentes) [reply fer-git-practice/index.js]: finish

File: fer-git-practice/index.html (hunk 2 above)
Comment (mframirezfuentes) [reply fer-git-practice/index.html]: finish

**Review by RickyCode — APPROVED** [review]

**Review by arielarmijo — APPROVED** [review]
Nice work

**Review by carrizotamara — APPROVED** [review]

**Pull request merged**